go/obscuronode/enclave/rpc/server.go

```diff
@@ -0,0 +1,152 @@
+package rpc
```

Comment:
is this file generated?

If yes, I'd suggest we move all the generated stuff to a subpackage. Not sure what the best practice in go is, but it'd be good to know exactly what is generated.

Reply:
This specific file isn't generated, but some are. I'll move the generated files to their own package.